src/Controller/Ajax.php

```diff
@@ -273,6 +276,16 @@ private function getActiveModel()
      */
     protected function exitScript()
     {
+        // TODO START give the response back or use own entry point for ajax call.
+        // The problem is the session will be update by event kernel.response.
+        $session = System::getContainer()->get('session');
+        $sessionBag = $session->getBag('contao_backend')->all();
+
+        $user = BackendUser::getInstance();
+
+        Database::getInstance()->prepare('UPDATE tl_user SET session=? WHERE id=?')
```

Comment:
Isn't this done in `BackendUser::__destruct()`?

Reply:
The magic method destruct is removed.

Before:
https://github.com/contao/core-bundle/blob/7c452603ae23038b0cd42a54f7b64a494b476a46/src/Resources/contao/classes/BackendUser.php#L112

After:
https://github.com/contao/core-bundle/blob/7b1d8995253edecdd71a11ee4d9eda1f861b009a/src/Resources/contao/classes/BackendUser.php#L112

Reply:
See above.

Reply:
See above.